ORLib/Package.swift

```diff
@@ -0,0 +1,34 @@
+// swift-tools-version:5.3
+import PackageDescription
+
+let package = Package(
+    name: "ORLib",
+    platforms: [
+        .iOS(.v14)
+    ],
+    products: [
+        .library(
+            name: "ORLib",
+            targets: ["ORLib"]),
+    ],
+    dependencies: [
+        // No external dependencies required
+    ],
+    targets: [
+        .target(
+            name: "ORLib",
+            dependencies: [],
+            path: "ORLib",
+            exclude: ["Info.plist"],
```

Comment:
Why this exclude ?
I don't have an Info.plist in ORLib and this generates a warning in XCode (referencing the ORLib package in the GenericApp).